core/domain/draft_upgrade_services_test.py

```diff
@@ -93,22 +93,65 @@ def test_try_upgrade_failure_due_to_unimplemented_upgrade_methods(self):
             draft_upgrade_services.try_upgrading_draft_to_exp_version(
                 self.DRAFT_CHANGELIST, 1, exploration.version, self.EXP_ID))
 
-    def test_try_upgrade_success(self):
+
+class DraftUpgradeUtilUnitTests(test_utils.GenericTestBase):
+    """Test the DraftUpgradeUtil module."""
+
+    # EXP_ID and USER_ID used to create default explorations.
+    EXP_ID = 'exp_id'
+    USER_ID = 'user_id'
+
+    # Creates and saves a default exploration.
+    def setUp(self):
+        super(DraftUpgradeUtilUnitTests, self).setUp()
+        self.save_new_valid_exploration(self.EXP_ID, self.USER_ID)
+
+    def helper_to_test_schema_verison_updates(
+            self, current_schema_version, schema_version_to_upgrade,
+            draft_change_list):
+        """Helper that uses the try_upgrading_draft_to_exp_version public
+        method to call the corresponding private method to update the draft
+        change list schema. For example, if the current_schema_version is 29
+        and the schema_version_to_upgrade is 30 then this helper would use the
+        try_upgrading_draft_to_exp_version public method to return the new
+        draft change list that has been updated according to the
+        _convert_states_v29_dict_to_v30_dict private method.
+
+        Args:
+            current_schema_version: string. The current schema version of the
+            draft change list (eg. '29')
+            schema_version_to_upgrade: string. The schema version to upgrade
+            the draft change list to (eg. '30')
+            draft_change_list: list(ExplorationChange). The list of
+                ExplorationChange domain objects to upgrade.
+
+        Returns:
+            list(ExplorationChange). The converted draft_change_list.
+        """
+
+        # Create an exploration change list with the command that will suggest
+        # migrating the schema from current_schema_version to
+        # schema_version_to_upgrade.
+        exp_migration_change_list = [exp_domain.ExplorationChange({
+            'cmd': exp_domain.CMD_MIGRATE_STATES_SCHEMA_TO_LATEST_VERSION,
+            'from_version': current_schema_version,
+            'to_version': schema_version_to_upgrade,
+            })]
```

Comment:
Deindent by 4 to match opening bracket indentation.

Reply:
Thanks!